### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by vianna-caterina**

### CODE REVIEW

File: staff/caterina-vianna/challenges/calculator-app/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Calculator App
+
+## Intro
+
+Blah Blah
+
+![Alt Calculator](./screenshot.png "Calculator App")
+
+## How it works
+
```
Comment (manuelbarzi) [inline staff/caterina-vianna/challenges/calculator-app/README.md:10]: description?

File: staff/caterina-vianna/challenges/calculator-app/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Calculator App
+
+## Intro
+
+Blah Blah
```
Comment (manuelbarzi) [inline staff/caterina-vianna/challenges/calculator-app/README.md]: blah blah?

File: staff/caterina-vianna/challenges/calculator-app/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## How it works
+
+## Technical description
+
+- index.html
+
+## Resources
+
+- style.css
```
Comment (manuelbarzi) [inline staff/caterina-vianna/challenges/calculator-app/README.md]: # warning!

- style.css should be in the technical description, not here

File: staff/caterina-vianna/challenges/calculator-app/README.md (hunk 2 above)
Comment (betoDavila86) [inline staff/caterina-vianna/challenges/calculator-app/README.md]: Add some description, please :)

File: staff/caterina-vianna/challenges/tic-tac-toe-app/style.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  }
+
+  .x::after {
+    font-size: 150px;
+  }
+
+  .o::after {
+    font-size: 175px;
+  }
+}
```
Comment (betoDavila86) [inline staff/caterina-vianna/challenges/tic-tac-toe-app/style.css:111]: I can see this last '}' redundant.

**Review by betoDavila86 — CHANGES_REQUESTED** [review]

File: staff/caterina-vianna/challenges/tic-tac-toe-app/tic_tac_toe.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```
Comment (manuelbarzi) [inline staff/caterina-vianna/challenges/tic-tac-toe-app/tic_tac_toe.html:1]: rename file to index.html

**Pull request closed without merging**